apps/bsd/src/screens/LoadElectionScreen.tsx

```diff
@@ -25,6 +25,7 @@ const LoadElectionScreen: React.FC<Props> = ({ setElection }) => {
     isLiveMode: boolean
     locales?: string
   }>()
+  const [isLoadingTemplates, setLoadingTemplates] = useState(false)
```

Comment:
`setIsLoadingTemplates`

(sorry, on mobile)

Reply:
I actually disagree with this. In my experience, getters and setters for boolean values typically follow the `isFoo`/`setFoo` pattern. I have had a hard time finding examples for or against this with React hooks specifically (here's [`setToggled`](https://dmitripavlutin.com/react-usestate-hook-guide/) and here's [`setIsToggled`](https://dev.to/alexkhismatulin/update-boolean-state-right-with-react-hooks-3k2i)), but in other contexts like Java [the idiom is quite strongly `setFoo`](https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter).

As with most conventions, I'm okay with either as long as we're consistent. Have a preference, @benadida & @carolinemodic?

Reply:
My sense is the convention we've been using leads to setIsLoadingTemplates, as in the setter is set + gettername. But I do find it a bit odd and I would support the other approach. 

I'm also not one who feels particularly strong about strict naming conventions. So how's that for an unhelpful comment? :)

Reply:
Merged this as-is for now, but let's revisit naming at some point.